data/events/2024-nairobi.yml

```
@@ -0,0 +1,104 @@
+
+startdate:  2024-06-01T08:00:00+00:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate:  2024-06-01T18:00:00+00:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start:  # start accepting talk proposals.
+cfp_date_end:  # close your call for proposals.
+cfp_date_announce:  # inform proposers of status
+
+cfp_link: "" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+
+registration_date_start: # start accepting registration. Leave blank if registration is not open yet. This will make the "Register" button appear on your "Welcome" page.
+registration_date_end: # close registration. Leave blank if registration is not open yet. If you set "registration_date_start" you need a value here.
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "Nairobi" # Defaults to city, but you can make it the venue name.
+#
+location_address: "" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+ # - name: propose
+ # - name: location
+ # - name: registration
+ # - name: program
+ # - name: speakers
+  - name: sponsor
+  - name: contact
+  - name: conduct
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Maamun Bwanakombo"
+    github: "https://github.com/Nabhany"
+    linkedin: "https://www.linkedin.com/in/maamun-bwanakombo-58849b141/"
+    image: ""
+  - name: "Kadima Samuel"
+    github: "https://github.com/kadimasum"
+    linkedin: "https://www.linkedin.com/in/kadima-samuel-804103bb/"
+    image: ""
+  - name: "Mercyline Nyansarola"
+    github: ""
+    linkedin: "https://www.linkedin.com/in/mercyline-nyansarola/"
+    image: ""
+  - name: "Emma Nyakio"
+    github: ""
+    linkedin: "https://www.linkedin.com/in/emma-nyakio/"
+    image: ""
+  - name: "Lawrence Juma" 
+    github: ""
+    linkedin: ""
+    image: ""
+  - name: "Ivy Jeptoo"
+    github: "devopsdays"
```

This is our github :) If Ivy has an account, please add it
```suggestion
    github: ""
```

---

Fixed in https://github.com/devopsdays/devopsdays-web/pull/13542/commits/1c0018a8a3bc0d24f2c3afd5653dd638712c615f